articles/basics-of-convolution-neural-networks/index.md

```diff
@@ -0,0 +1,168 @@
+
+#### Pooling layer
+
+A pooling layer is a form of non-linear downsampling that reduces the dimensionality of the feature maps of an input image, making its representation smaller and more manageable. This process reduces the number of parameters and computation resources in the network.
+
+It is important to note that a 2 x 2 filter size applied with a stride of 2 is the most used in CNN.
+
+##### Max Pooling
+
+**Max pooling** is the most popular among the pooling operations. It extracts patches from the input feature maps, picks the largest value on each patch, and discards the rest.
+
+Below is an example of a max-pooling operation. It consists of a filter size of 2 x 2 that extracts a 2 x 2 patch from the input tensor. It outputs the largest value in each patch. Max-pooling results in the downsampling of the input by a factor of 2.
+
+![Max Pooling](/engineering-education/basics-of-convolution-neural-networks/max-pooling.PNG)
+
+##### Average Pooling
+
+Besides, **average pooling** is another type of pooling operation that's worth mentioning. It performs downsampling by performing an average of all values in a feature map. Thus, picking this averaged value as the new value of the feature space. 
+
+![Average Pooling](/engineering-education/basics-of-convolution-neural-networks/average-pooling.png)
+
+#### Fully Connected layer
+
+The Fully Connected (FC) layer is the last building block in a Convolution Neural Network. 
+The output feature maps of the layer before the FC layer are transformed into a one-dimension (1D) array of numbers, i.e., flattened. That layer is connected to one or more fully connected layers. 
+
+The FC layer maps the extracted features from the convolution and pooling layers onto the final output, i.e., classification.
+
+![A fully connected layer with a flattened input](/engineering-education/basics-of-convolution-neural-networks/fully-connected.PNG)<br>
+
+*[Image Source: Stanford](https://stanford.edu/~shervine/teaching/cs-230/cheatsheet-convolutional-neural-networks)*
+
+### Applications of Convolution Neural Networks
+
+#### Computer Vision
+
+Image classification is a very common problem in computer vision. The goal in image classification is to learn features directly from image data. These learned features are then used for classification. For instance, feeding an image of a cat as our input, the network should classify the image as a cat. 
+
+Recently, CNN has been used for image captioning. Here, the goal is to generate a sentence that describes the semantic content in an image. It is achieved by combining CNN with a [Recurrent Neural Network](/https://stanford.edu/~shervine/teaching/cs-230/cheatsheet-recurrent-neural-networks/)(RNN). In the last layer in the CNN architecture, instead of a fully connected (FC) layer, the last layer of the CNN is replaced with an RNN.   
```

Comment:
I feel some more explanation is required here. A beginner will find it difficult to understand how an RNN is getting fixed to a CNN.